### ISSUE TITLE
Yandex with custom domain doesn't work

### ISSUE THREAD

**Issue by sinnlosername** [Foundry376/Mailspring#1337]
##### What operating system are you using?
Windows

##### What version of Mailspring are you using?
Latest

##### Is this an email sync or connection issue?
Yes

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
Yes, Yandex

##### Is the issue reproducible with a particular attachment, message, signature, etc?
Normally when trying to login mailspring seems to take my email "name@yandex.ru" but it only uses the "name" in the IMAP command. This means it tries to do `login name <password>`. 

However it does the same when using a custom domain, which is wrong. For example my custom email is admin@my.tld, then it will still try to login using `login admin <password>`, while (I think) it should do `login admin@my.tld <password>`

This issue makes it impossible to login using a custom domain

**Comment by IlyaSemenov** [comment]
I am having the same problem, here's the screenshot for clarity:

<img width="1012" alt="screenshot 2019-01-26 at 19 38 53" src="https://user-images.githubusercontent.com/128121/51787268-2f178580-21a2-11e9-9aa1-17b8784e7ba7.png">

and here's the log:

```
Important:You may need to link a phone number to your Yandex account before connecting email apps. If you use two-factor auth, you need to create an App Password for Mailspring.
https://phone-passport.yandex.com/phones

----------IMAP----------
* OK Yandex IMAP4rev1 at imap65j.mail.yandex.net:993 ready to talk with ::ffff:176.51.101.106:49766, 2019-Jan-26 15:40:48, meX427uTiuQ1
1 CAPABILITY
* CAPABILITY IMAP4rev1 CHILDREN UNSELECT LITERAL+ NAMESPACE XLIST BINARY UIDPLUS ENABLE ID AUTH=PLAIN AUTH=XOAUTH2 IDLE MOVE
1 OK CAPABILITY Completed.
2 LOGIN ilya *********
2 NO [AUTHENTICATIONFAILED] LOGIN Invalid credentials or IMAP is disabled sc=meX427uTiuQ1_261240_65j
```

**Comment by bengotow** [comment]
Hey thanks for reporting this! I'll see if I can get this fixed in the next release.

**Comment by bengotow** [comment]
Hey! This has been fixed in 1.6.0 which shipped today. The app should auto-update, but you can also get the new version manually from https://getmailspring.com/download. Thanks - enjoy the app!

**Comment by IlyaSemenov** [comment]
Thanks. Actually, it was fixed in 1.5.6: https://github.com/Foundry376/Mailspring/releases/tag/1.5.6

I had successfully added an another custom Yandex domain a week ago with 1.5.7.
